app/views/api/v2/hosts/show.json.rabl

```diff
@@ -25,3 +25,16 @@ end
 child :config_groups do
   extends "api/v2/config_groups/main"
 end
+
+host_additional_tabs(@host).each do |id, tab|
```

Comment:
Should the index view also show the information from aspects (I know it's pretty basic atm)?